README.md

```diff
@@ -26,11 +26,23 @@ Or install it yourself as:
 
 ## Usage
 
-This gem requires 2 columns for each table: `parentry` and `parent_id`. A sample migration would look like:
+This gem requires 2 columns for each table: `parentry` and `parent_id`, and is dependent on the `ltree` postgresql extension. A sample migration would look like:
```

Comment:
`ltree` is actually not a requirement. You can persist the hierarchy as an array if you can't or don't want to use the `ltree` module. 

```ruby
class MyTable < ApplicationRecord
  include Parentry
  parentry strategy: :array
end
```

I really need to spend some time on adding good documentation for this gem.